### PR TITLE
Mobile menu enhancement

### PR DESCRIPTION
**Pull request by mikeyarce**
handheld-sf
* Translation & Accessibility ready
* State resets when you close the handheld menu
* Transitions to match handheld menu

### CODE REVIEW

File: assets/sass/base/_base.scss [hunk 1]
```diff
@@ -1194,6 +1194,42 @@ button.menu-toggle {
...
+	clear: both;
+	
+	ul {
+		margin: 0;
+
+		&.menu {
+			li {
+				position: relative;
+				button:after {
+					font: normal normal normal 16px/1 FontAwesome;
```
Comment (jameskoster) [inline assets/sass/base/_base.scss]: Can use `@include fa-icon();` here.

File: assets/sass/base/_base.scss [hunk 2]
```diff
@@ -1194,6 +1194,42 @@ button.menu-toggle {
...
+					content: "\f107";
+				}
+			}
```
Comment (jameskoster) [inline assets/sass/base/_base.scss]: Add a return

File: assets/sass/base/_base.scss [hunk 3]
```diff
@@ -1194,6 +1194,42 @@ button.menu-toggle {
...
+			.sub-menu {
+				visibility: hidden;
+				opacity: 0;
+				max-height: 0;
+				transition: all,ease,.9s;
```
Comment (jameskoster) [inline assets/sass/base/_base.scss]: Add a return

**Comment by mikeyarce** [comment]
@jameskoster sounds good!  Added that and your other notes as well.

File: assets/sass/base/_base.scss [hunk 4]
```diff
@@ -1204,22 +1209,26 @@ button.menu-toggle {
...
 						transform: rotate(180deg);
 					}
 				}
 			}
+
 			.sub-menu {
 				visibility: hidden;
 				opacity: 0;
 				max-height: 0;
 				transition: all,ease,.9s;
```
Comment (jameskoster) [inline assets/sass/base/_base.scss]: Spaces after commas xD

**Comment by jameskoster** [comment]
Just one comment, looking good man :)

**Comment by mikeyarce** [comment]
Spaces added!

**Comment by jameskoster** [comment]
*Ping* @tiagonoronha go ahead and merge if you're happy :)

**Comment by GlennMartin1** [comment]
Any chance this could be moved to 2.2.7?

**Comment by tiagonoronha** [comment]
No, this PR needs work to drop the jQuery dependency. We removed the dependency in 2.2.6.

**Comment by tiagonoronha** [comment]
Jus refactored this PR. :)

@mikejolley please test and let me know how it goes!

File: inc/class-storefront.php [hunk 5]
```diff
@@ -250,6 +250,16 @@ public function scripts() {
 			wp_enqueue_script( 'storefront-navigation', get_template_directory_uri() . '/assets/js/navigation' . $suffix . '.js', array(), $storefront_version, true );
 			wp_enqueue_script( 'storefront-skip-link-focus-fix', get_template_directory_uri() . '/assets/js/skip-link-focus-fix' . $suffix . '.js', array(), '20130115', true );
 
+			$storefront_l10n = array();
+
+			if ( has_nav_menu( 'handheld' ) ) {
+				$storefront_l10n['expand']         = __( 'Expand child menu', 'storefront' );
```
[the remark this answers is not in the document]
Comment (tiagonoronha) [reply inc/class-storefront.php]: https://github.com/woocommerce/storefront/pull/669/commits/14fb73f1ed9dc4d7a79c5b8e2a9dee51f31fe058

**Comment by tiagonoronha** [comment]
Kudos @mikeyarce 🎉

**Pull request merged**